ground/src/main/java/com/google/android/ground/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -146,46 +128,36 @@ class GoogleMapsFragment : Hilt_GoogleMapsFragment(), MapFragment {
     layoutInflater: LayoutInflater,
     viewGroup: ViewGroup?,
     bundle: Bundle?
-  ): View =
-    super.onCreateView(layoutInflater, viewGroup, bundle).apply {
+  ): View {
+    Timber.v("Lifecyle event: onCreateView()")
```

Comment:
Please cleanup debug only logs.

Reply:
All lifecycle events are logged across the app. Ok to keep here in that case?